web/src/beta/lib/core/StoryPanel/Page/index.tsx

```diff
@@ -32,8 +45,19 @@ type Props = {
     vt?: ValueType,
     v?: ValueTypes[ValueType],
   ) => Promise<void>;
+  handleMoveBlock: (id: string, targetId: number, blockId: string) => void;
 };
 
+const StoryPageContext = createContext({
+  isHovered: false,
+  handleOnMouseOut: () => {},
+  handleOnMouseOver: (e: MouseEvent<HTMLDivElement>) => {
+    e;
+  },
+});
+
+export const useStoryPageContext = () => useContext(StoryPageContext);
```

Comment:
Can you give any big reasons you are using context here?
For one, I feel like it adds complexity since most things we need downstream we are passing as props (even if they need to go far).
But also, I think because you define and use hover state here at the page level, if I hover over one block, I get ALL handles showing in that page.

Reply:
@KaWaite I removed this